cg/apps/tb/api.py

```diff
@@ -49,6 +52,36 @@ def parse_qcmetrics(data: dict) -> dict:
         """Call internal Trailblazer MIP API."""
         return files.parse_qcmetrics(data)
 
+    def is_analysis_ongoing(self, case_id: str) -> bool:
```

Comment:
Great! First step to decouple 😄

Reply:
yes, starting that long winding road.... 😄